### ISSUE TITLE
Raw IMU Message Delay

### ISSUE THREAD

**Issue by TroyeFun** [guardstrikelab/carla_apollo_bridge#78]
I met error messages as below when running apollo-carla co-simulation following the steps in Readme.
```
You haven't selected a vehicle yet!
GPS trigers safe mode.
Raw IMU Message Delay. IMU message is 168126848847 cycle 1.68127e+09 sec behind current time.
Raw GPS Message Delay. GPS message is 168126848847 cycle 1.68127e+09 sec behind current time.
```
And the lidar point-cloud shown is Dreamview seems not right, either.
![Screenshot from 2023-04-12 12-05-21](https://user-images.githubusercontent.com/24251746/231347551-375f9f35-e5ae-452e-89c0-e759bec64a59.png)

The page of module controller is shown below. I cannot open the GPS and Velodyne module, and the status of GPS is always FATAL or ERROR.
![Screenshot from 2023-04-12 12-01-10](https://user-images.githubusercontent.com/24251746/231347837-bd783514-c403-48ac-9377-5528cdea632a.png)

By the way, the vehicle can run automatically, though often hitting obstacles road-side.

This problem has been reported in #30 , but it hasn't been solved yet. Can anyone help? Thanks a lot!

**Comment by sweartoyou** [comment]
same question, did you solve the problem?

**Comment by TroyeFun** [comment]
No, it seems that the message delay problem doesn't matter. But the problem of lidar point-cloud does trouble me.

**Comment by WilliaJing** [comment]
@TroyeFun GPS of Components is used to monitor the status of the actual vehicle hardware GNSS,because we are simulating and sending cyber messages directly,so GPS is always FATAL.

**Comment by SEEKERXC** [comment]
Hello!
This issue is due to an incorrect timestamp in the header of the IMU message.
This issue does not affect running, but we still fixed it.
```
imu_msg.header.timestamp_sec = cyber_time.Time.now().to_sec()
```
We will soon release it.

**Comment by SEEKERXC** [comment]
Problem solved by [Release 0.3.0](https://github.com/guardstrikelab/carla_apollo_bridge/releases/tag/0.3.0), so this issue is closed.

**Comment by AOOOOOA** [comment]
I also met the same problem even using the latest carla_apollo_bridge version. @XiaoFei9704 @WilliaJing
